README.md

```diff
@@ -1,20 +1,223 @@
+```
+
+If you want to be sure you are getting the latest commits when developing,
+clearing composer cache, deleting the folders and re-running composer update
+seems to be a solid approach:
+
+```bash
+
+rm -rf web/profiles/contrib/ web/modules/contrib/ web/themes/contrib/;
+composer clear-cache; composer update --with-dependencies --no-cache;
+lando start;
+lando drush si localgov_microsites -y;
+
+```
+
+If you run into [memory limit errors](https://getcomposer.org/doc/articles/troubleshooting.md#memory-limit-errors)
+when running Composer commands, prefix the commands with `COMPOSER_MEMORY_LIMIT=-1`.
+For example, to install the project run:
+
+```bash
+COMPOSER_MEMORY_LIMIT=-1 composer create-project localgovdrupal/localgov-microsites-project MY_PROJECT
+```
+
+## Contributing
 
-This node includes layout paragraphs and paragraphs to demonstrate some of the components for a new microsite. When a new microsite is created, it attempts to clone this node into the new microsite. 
+See [CONTRIBUTING.md](CONTRIBUTING.md) for current contribution guidelines.
+
+## Issue tracking
+
+Most issues will be tracked in this repository
+<https://github.com/localgovdrupal/localgov_microsites/issues>.
+
+Development issues relating to specific projects or module should be tracked in
+the project repository.
+
+## Development setup
+
+The main development environment in use is currently
+[Lando](https://docs.lando.dev/) – a Docker based development environment that
+works on Linux, MacOS and Windows.
+
+@todo Document Lando setup.
+
+## Coding standards
+
+PHP CodeSniffer is installed as a dev dependency by Composer and configured to
+use Drupal coding standards and best practices. It is a good idea to run these
+before committing any code. All code in pull requests should pass all
+CodeSniffer tests.
+
+To check code using Lando run:
+
+```bash
+lando phpcs
+```
+
+To attempt to automatically fix coding errors in Lando run:
+
+```bash
+lando phix
+```
+
+### Coding standards resources
+
+* [Drupal coding standards](https://www.drupal.org/docs/develop/standards)
+
+## Running tests
+
+The included `phpunit.xml.dist` file contains configuration for automatically
+running the LocalGov Drupal test suite.
+
+To run all LocalGov Drupal tests with Lando use:
+
+```bash
+lando phpunit
+```
+
+To run all the tests for a specific module use:
+
+```bash
+lando phpunit web/modules/contrib/localgov_my_module
+```
+
+Tests can be filtered using the `--filter` option. To only run a specific test
+use:
+
+```bash
+lando phpunit --filter=myTestName
+```
+
+### Testing resources
+
+* [PHPUnit documentation](https://phpunit.readthedocs.io/en/7.5/)
+* [Drupal 8 PHPUnit documentation](https://www.drupal.org/docs/8/testing/phpunit-in-drupal-8)
+* [Drupal 8 testing documentation](https://www.drupal.org/docs/8/testing)
+* [Workshop: Automated Testing and Test Driven Development in Drupal 8](https://github.com/opdavies/workshop-drupal-automated-testing)
+
+## Default content
```

Comment:
actually I think all this wants to move up in to the Developer notes for default content section above.

Reply:
any chance you could do that please?